Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputBindingPropertiesView.cs

```diff
@@ -102,11 +103,172 @@ protected override void DrawGeneralProperties()
                     }
                 }
 
+                // Show the specific controls which match the current path
+                DrawMatchingControlPaths();
+
                 // Control scheme matrix.
                 DrawUseInControlSchemes();
             }
         }
 
+        /// <summary>
+        /// Used to keep track of which foldouts are expanded.
+        /// </summary>
+        private static bool showMatchingLayouts = false;
+        private static Dictionary<string, bool> showMatchingChildLayouts = new Dictionary<string, bool>();
+
+        /// <summary>
+        /// Finds all registered control paths implemented by concrete classes which match the current binding path and renders it.
+        /// </summary>
+        private void DrawMatchingControlPaths()
+        {
+            var path = m_ControlPathEditor.pathProperty.stringValue;
+            if (path == string.Empty)
+                return;
+
+            var deviceLayoutPath = new InternedString(InputControlPath.TryGetDeviceLayout(path));
+            var parsedPath = InputControlPath.Parse(path).ToArray();
+
+            EditorGUILayout.BeginVertical();
+
+            // If the provided path is parseable into device and control components, draw UI which shows all control layouts that match the path.
+            bool matchExists = false;
+            showMatchingLayouts = EditorGUILayout.Foldout(showMatchingLayouts, "Matched Controls");
+
+            if (showMatchingLayouts)
+            {
+                if (parsedPath.Length >= 2)
+                {
+                    if (deviceLayoutPath != InputControlPath.Wildcard)
+                    {
+                        var rootDeviceLayout = EditorInputControlLayoutCache.allLayouts.FirstOrDefault(x => !x.isOverride && !x.hideInUI && x.name == deviceLayoutPath);
+                        if (rootDeviceLayout != null)
+                        {
+                            matchExists |= DrawMatchingControlPathsForLayout(rootDeviceLayout, in parsedPath, true);
+                        }
+                    }
+                    else
+                    {
+                        var genericDeviceLayouts = EditorInputControlLayoutCache.allLayouts
+                            .Where(x => x.isDeviceLayout && !x.hideInUI && !x.isOverride && x.isGenericTypeOfDevice && x.baseLayouts.Count() == 0).OrderBy(x => x.displayName);
+
+                        foreach (var genericDeviceLayout in genericDeviceLayouts)
+                        {
+                            matchExists |= DrawMatchingControlPathsForLayout(genericDeviceLayout, in parsedPath);
+                        }
+                    }
+                }
+
+                // Otherwise, indicate that no layouts match the current path.
+                if (!matchExists)
+                {
+                    EditorGUILayout.LabelField("No registered control paths match this current binding");
```

Comment:
Could be valuable to add a tooltip to this label as some binding control paths may only be registered at runtime so it may match at that time.

Reply:
Done